tests/prepared_statment.cc

```diff
@@ -75,6 +75,28 @@ int main() {
 			(pps4 << test << test << test)++;
 		}
 
+		{
+			auto prep = db << "select ?";
+
+			prep << 5;
+
+			prep.execute();
+			try {
+				prep.execute();
```

Comment:
I think you want to add a `exit(EXIT_FAILURE);` here to handle the case that the error is not detected.